homeassistant/components/knx/schema.py

```diff
@@ -400,14 +422,11 @@ class LightSchema:
             }
         ),
         vol.Any(
-            # either global "address" or all addresses for individual colors are required
+            # either global "address" "individual_colors" is required
```

Comment:
Please rephrase this comment; it's difficult to understand with an _either_ without an _or_